meerk40t/core/elements.py

```diff
@@ -6506,6 +6681,18 @@ def reset_user_changes(node, copies=1, **kwargs):
         def merge_elements(node, **kwargs):
             self("element merge\n")
 
+        @self.tree_conditional(lambda node: node.lock)
+        @self.tree_separator_before()
+        @self.tree_operation(_("Unlock Element"), node_type=elem_nodes, help="")
+        def element_unlock_manipulations(node, **kwargs):
+            self("element unlock\n")
+
+        @self.tree_conditional(lambda node: not node.lock)
+        @self.tree_separator_before()
+        @self.tree_operation(_("Lock manipulations"), node_type=elem_nodes, help="")
```

Comment:
Non-parallel wording. Unlock/Lock (Element/manipulations) -- One way or the other on the noun.

Reply:
Amended